packages/react/src/components/Skeleton/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Circle as SkeletonCircle, type CircleProps } from './Circle/Circle';
+import {
+  Rectangle as SkeletonRectangle,
+  type RectangleProps,
+} from './Rectangle/Rectangle';
+import { Text as SkeletonText, type TextProps } from './Text/Text';
+
+/**
+ * Represent a draft of page while the content loads. Mix different skeleton components to create your layout.
+ * @example
+ *   <Skeleton.Circle />
+ *   <Skeleton.Text />
+ *   <Skeleton.Text />
+ *   <Skeleton.Text />
+ *   <Skeleton.Rectangle />
+ */
+type SkeletonComponent = {
+  /**  Skeleton component used for indicating loading elements of circular shape */
+  Circle: typeof SkeletonCircle;
+  /**  Skeleton component used for indicating loading elements of rectangle shape */
+  Rectangle: typeof SkeletonRectangle;
+  /**  Skeleton component used for indicating loading elements of text */
+  Text: typeof SkeletonText;
+};
+
+const Skeleton: SkeletonComponent = {
+  Circle: SkeletonCircle,
+  Rectangle: SkeletonRectangle,
+  Text: SkeletonText,
+};
+
+export type { CircleProps, RectangleProps, TextProps };
```

Comment:
I think maybe we should export this as `Skeleton...Props`

Reply:
Uh.. I don't really understand what you mean here 😅

Reply:
Rename the props to `SkeletonCircleProps`, `SkeletonRectangleProps` and `SkeletonTextProps`. We might as well rename it everywhere it is used, to be consistent 😄

Reply:
This is important because when we export them in the root all components and types are on the same level, so we need to prefix them with the name to where they belong :)

Reply:
Oh yeah! That's a good point!
I'll change it! 😊